Java/core/src/main/java/com/amazon/randomcutforest/RandomCutForest.java

```diff
@@ -815,6 +817,36 @@ public DensityOutput getSimpleDensity(double[] point) {
      *                              missing values.
      * @return A point with the missing values imputed.
      */
+    public ArrayList<double[]> getSimpleConditionalField(double[] point, int numberOfMissingValues,
```

Comment:
I think we need a better name for this method, but we can do it in another PR.

#151